cvmfs/repository.py

```diff
@@ -141,11 +28,10 @@ class Repository(object):
 
     def __init__(self, fetcher):
         self._fetcher = fetcher
-        self._opened_catalogs = {}
+        self.opened_catalogs = {}
```

Comment:
It is still used in the _opened_catalogs_for_path method, but precisely now we are modifying it